src/main/java/club/issizler/okyanus/api/json/click/RunConsumer.java

```diff
@@ -0,0 +1,63 @@
+package club.issizler.okyanus.api.json.click;
+
+import club.issizler.okyanus.api.Okyanus;
+import club.issizler.okyanus.api.cmdnew.Command;
+import club.issizler.okyanus.api.cmdnew.CommandOf;
+import club.issizler.okyanus.api.cmdnew.req.CommandSenderReq;
+import club.issizler.okyanus.api.cmdnew.req.CommandSenderType;
+import club.issizler.okyanus.api.entity.Player;
+import club.issizler.okyanus.api.json.JsonEvent;
+import org.cactoos.Proc;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+import static club.issizler.okyanus.api.cmdnew.CommandOptions.*;
+
+public final class RunConsumer implements JsonEvent {
+
+    private final UUID uuid = UUID.randomUUID();
+    private final Command command;
+
+    public RunConsumer(boolean removeAfter, @NotNull final Proc<Player> proc) {
+        command = new CommandOf(
+            "uuid",
```

Comment:
Wouldn't having multiple commands with the same ID cause some sort of collision?

Reply:
Actually, no. Our command system is checking just 'label' of the first CommandOf.